manifests/r/RejminetGroupInc/PDFShow/1.0.3/RejminetGroupInc.PDFShow.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: RejminetGroupInc.PDFShow

```

```suggestion
PackageIdentifier: IanDarwin.PDFShow
```
Since this package is released through GitHub, it would be better to use the repository owner instead of the publisher. Remember when changing this value you will have to change the folder the manifest is in